modules/date-localizer/main/index.coffee

```diff
@@ -0,0 +1,168 @@
+class DateLocalizer
```

Comment:
Somewhat contrary to its name, this localizes both Dates and Times. Might it be worth calling it DateTimeLocalizer or Localizer or similar ? (Related to #18, but you might want to do that in a separate module)